blazor/chart/chart-print.md

```diff
@@ -373,6 +373,134 @@ If you set `IsBase64` parameter to **true** while calling the [ExportAsync](http
 
 ```
 
+### BeforeExport ExcelProperties
+
+The ExcelProperties class in the `Exporting` event allows users to customize the exported Excel sheet by modifying the properties of rows, columns, and cells before the file is generated.
```

Comment:
Is ExcelProperties class available in the below code?
The content does not corresponds to the code used.
Only Excel export is explained. When will happen to other export types when `Workbook` property is not provided in notes.
The properties, Cancel, Height and Width, in the ChartExportingEventArgs is not explained.